.changeset/mighty-dryers-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sl-design-system/grid": patch
+---
+
+fixed checkbox checked value
```

Comment:
Please the line with a capital letter (this text will end up in the `CHANGELOG.md` file after release). Please elaborate a bit more what is fixed :)

Reply:
Added more info